styles/style.css

```diff
@@ -6,3 +6,265 @@ Green: #00B172
 White: #FFF
 
 */
+/* 
+===================================================
+CLEARING BROWSER DEFAULTS AND SETTING GLOBAL STYLES
+===================================================
+ */
+body {
+  margin: 0;
+  font-size: 1.5rem;
+  font-weight: lighter;
+  color: #1a1a1a;
+}
+
+/* 
+==========================================
+LAYOUT
+==========================================
+ */
+
+section,
+header {
+  border: 1px red solid;
+}
+.row,
+col {
+  border: 1px green dashed;
+}
+/* layout guides, to be used in development only */
+nav {
+  width: 100%;
+  height: 4.5rem;
+  background-color: #fff;
+}
+.logo {
+  width: 10%;
+  margin: 1rem 0 0 1.5rem;
+}
+.hamburger {
+  display: none;
+  padding: 15px;
+  cursor: pointer;
+  #mobile_menu {
+    color: #00b172;
+  }
+}
+.off {
+  display: none;
+}
+.on {
+  display: block;
+  background-color: #fff;
+  padding-bottom: 0.5rem;
+  margin: 0 0 0 0;
+}
+ul {
+  list-style-type: none;
+  display: flex;
+  float: right;
+}
+li {
+  font-size: 1.5rem;
+  margin: 0.5rem 2rem 0 0;
+  font-weight: light;
+}
+
+.landing {
+  min-height: 30rem;
+  background-image: url("../images/landing.jpg");
+  background-size: contain;
+}
+.landing span {
+  position: absolute;
+  top: 40%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: #fff;
+  text-align: center;
+}
+.landing h1 {
+  font-size: 3rem;
+}
+.landing p {
+  font-weight: 100;
+}
+
+.features {
+  min-height: 25rem;
+  margin: 2rem 0 2rem 0;
+}
+.features h2 {
+  text-align: center;
+  text-decoration: underline #00b172;
+  text-underline-position: under;
+  margin-bottom: 2em;
+}
+.features h3 {
+  color: #00b172;
+}
+.features .col-1 {
+  text-align: center;
+}
+.features article {
+  margin: 1rem 0 2rem 0;
+}
+
+.features .row article img {
+  width: 20%;
+}
+.features .row {
+  justify-content: space-evenly;
+  margin: 0 0 3rem 0;
+}
+
+.app {
+  width: 95%;
+  background-color: #00b172;
+  margin: 0 auto;
+  color: #fff;
+}
+.app main {
+  margin: 1rem 1rem 1rem 1rem;
```

Comment:
Can just be margin: 1rem;

Reply:
Ohkay.
I originally thought one had to set  margins for each dimension.
Noted 🙂